testing/unittests/jlibtests.cpp

```diff
@@ -289,14 +289,14 @@ class JlibTraceTest : public CppUnit::TestFixture
         {
             OwnedSpanScope serverSpan = queryTraceManager().createServerSpan("failedErrorSpanEscaped", emptyMockHTTPHeaders);
             SpanError * error = new SpanError("hello");
```

Comment:
error leaks.  Simpler (as you have done elsewhere) to use a local variable instead.

same below line 299